tools/BUILD

```diff
@@ -0,0 +1,42 @@
+load(
+    "//:python.bzl",
+    "gematria_py_binary",
+)
+
+package(
+    default_visibility = ["//visibility:private"],
+)
+
+PACKAGE_SHARED_OBJECT_DEPS = [
+    "//gematria/llvm/python:canonicalizer.so",
+    "//gematria/llvm/python:llvm_architecture_support.so",
+    "//gematria/basic_block/python:basic_block_protos.so",
+    "//gematria/basic_block/python:basic_block.so",
+    "//gematria/datasets/python:bhive_importer.so",
+    "//gematria/granite/python:graph_builder.so",
+    "//gematria/model/python:oov_token_behavior.so",
+    "//:package_pybind11_abseil_status",
+]
+
+PACKAGE_PYTHON_DEPS = [
+    "//gematria/proto:basic_block_py_pb2",
+    "//gematria/proto:canonicalized_instruction_py_pb2",
+    "//gematria/proto:throughput_py_pb2",
+]
+
+genrule(
+    name = "shared_object_list",
+    srcs = PACKAGE_SHARED_OBJECT_DEPS,
+    outs = ["shared_object_list.txt"],
+    cmd = "echo {} > $@".format(" ".join(["$(rootpath {})".format(x) for x in PACKAGE_SHARED_OBJECT_DEPS])),
+)
```

Comment:
If we need this just to pass the list of files to build_pip_package.py, we might be able to skip this rule, and just iterate through the directory specified in the environment variable `RUNFILES_DIR` (which is set by Bazel's Python launcher).

Reply:
From what I understand of the documentation, you only get a `RUNFILES_DIR`  if you don't have a `RUNFILES_MANIFEST_FILE` (not sure how you decide what you get) and I'm getting a manifest file. Globbing through the directory should work, but we might need to do some path rewriting there so that `setuptools` understands what it's looking it. I think this is probably the most elegant solution to this specific problem, but I might not be seeing a better solution.